<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TpuzzleSolver</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_unitsImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_dstImg">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseUnitsImg">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>340</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>选择单元块图案</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_resultImg">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>321</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(167, 167, 167);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_chooseDstImg">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>340</y>
      <width>101</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>选择目标图案</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_solve">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>340</y>
      <width>87</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>求解</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
